### PR TITLE
Migrate Trilemma Triangle to Shadcn

### PR DESCRIPTION
**Pull request by minimalsm**
## Description

- Migrated Trilemma Triangle to Shadcn
- Add default border color from DS 

## Note

@nloureiro I took some liberties with picking colors: the current production Trilemma Triangle still uses blue for lightmode and orange for darkmode. This PR now uses purple instead, let me know if you have any suggestions/tweaks you'd like and I'll update.

## Related Issue

#13946

### CODE REVIEW

File: tailwind.config.ts
Comment (pettinarip) [inline tailwind.config.ts:160]: Curious about this, @nloureiro do we have a default color for our borders?

**Review by pettinarip — APPROVED** [review]
LGTM @minimalsm, will wait for a designers review before merging.

File: tailwind.config.ts
Comment (nloureiro) [reply tailwind.config.ts:160]: this was added for the homepage and we should use on other pages too
Screen Shot 2024-09-27 08 41 20 AM

Comment (nloureiro) [reply tailwind.config.ts:160]: Screen Shot 2024-09-27 08 42 18 AM

**Comment by nloureiro** [comment]
looks good to me!!

thank you @minimalsm :) 

Screen Shot 2024-09-27 08 51 26 AM

**Review by nloureiro — APPROVED** [review]

**Review by fernandoguadmendezespinoza17 — APPROVED** [review]

**Pull request merged**
